src/components/dialog/index.tsx

```diff
@@ -15,26 +16,43 @@ export default function Dialog({
 	isOpen,
 	onDismiss,
 	label,
+	variant = 'modal',
```

Comment:
new `variant` prop for dialog. I'm not sure what to call the default value....`base`? 🤷